specs/solid-indexing/index.html

```diff
@@ -88,7 +130,219 @@ <h3>Namespaces</h3>
   </section>
 
   <section>
```

Comment:
some intro text at the top of the vocabulary section would be nice... Currently it is very abrupt.

Reply:
Fixed in https://github.com/Wimmics/solid-start/pull/47/commits/b71e281de8267a11f748a96e324bf6176435a126